.github/workflows/PR-content-check.yml

```diff
@@ -25,10 +22,10 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha }}
           fetch-depth: 0
-          path: ion-java-new
+          path: ion-new-commit
 
       - name: Check the content of the last commit
         id: check-content
         run: |
-          cd ion-java-new
-          if [[ $(git log -1 --name-only) == *"src/"* ]]; then echo "result=pass" >> $GITHUB_OUTPUT; else echo "result=fail" >> $GITHUB_OUTPUT; fi
+          cd ion-new-commit
+          if [[ $(git log -1 --name-only) == *"src/"* ]] || [[ $(git log -1 --name-only) == *"ion/"* ]]; then echo "result=pass" >> $GITHUB_OUTPUT; else echo "result=fail" >> $GITHUB_OUTPUT; fi
```

Comment:
The test here could have false positives if you have a modified directory ending in `ion`.

```
$ if [[ $(echo -e "bastion/\nsrc/") == *"ion/"* ]]; then echo "True"; else echo "false"; fi
True
```

If that is a concern, you could pass the output of git's log through an egrep to make sure the start of the line is `ion/`, like:
```
$ if [[ $(echo -e "bastion/\nsrc/" | egrep "^ion/") ]]; then echo "True"; else echo "false"; fi
false
```

Reply:
Thanks for catching up this. I will update in the ion-python PR. For reference, I'll change it to below:

To match paths including `amazon/ion/`.
```
if [[ $(git log -1 --name-only | egrep "^amazon/ion/") ]]; then echo "result=pass" >> $GITHUB_OUTPUT; else echo "result=fail" >> $GITHUB_OUTPUT; fi
```